packages/grafana-data/src/datetime/datemath.ts

```diff
@@ -50,14 +52,41 @@ export function parse(
   if (!text) {
     return undefined;
   }
+  return toDateTime(text, { roundUp, timezone, fiscalYearStartMonth });
+}
+
+export interface ConversionOptions {
+  /**
+   * Set the time to endOf time unit, otherwise to startOf time unit.
+   */
+  roundUp?: boolean;
+  /**
+   * Only string 'utc' is acceptable here, for anything else, local timezone is used.
+   */
+  timezone?: TimeZone;
```

Comment:
deprecated import of `TimeZone` suggests to use newer from `import { TimeZone } from '@grafana/schema';
`